Python/Tkinter/Calculator/calculator.py

```diff
@@ -0,0 +1,122 @@
+# ===========================================================================
+# Title: calculator.py
+# Description: This program is a calculator
+# Author : Mathieu Caro
+# Source : Udemy
+# ===========================================================================
+
+# -----------------------------Import of modules------------------------------
+import tkinter as tk  # tkinter: package for standard python interface
+import tkinter.font as font
+from tkinter import ttk  # tkk: set of widgets:buttons, labels...
+# ----------------------------------Constants---------------------------------
+ORANGE = "#FF7900"
+BLACK = "#000000"
+WHITE = "#FFFFFF"
+BLUE = "#0090FC"
+# -----------------------------Global Variables--------------------------------
+equation=""
+# ---------------------------------Functions----------------------------------
+def show(value):
+    """This function enables to set the value of each button on screen"""
+    global equation
+    equation+=value #we add each button value to the string equation
+    label_result.set(equation) 
+    #for each additional button value it updates the value of the equation
+    
+def clear():
+    """This function simply clear the equation after using the "C" button"""
+    global equation
+    equation=""
+    label_result.set(equation)
+    
+def calculate():
+    """This function calculate from the string "equation" """
+    global equation
+    result=""
+    if equation !="":
+        try:
+            result=eval(equation)
+        #here eval enables from a string to transform into a calcul
+        #in that way, eval send back the result of the operation
+        except:
+            result = "error"
+            equation=""
+        #in case the operation is not calculable, there is a error
+        label_result.set(result)
+    
+def buttons_parameters(text_,width_,height_,foreground_color,backgroundcolor_,char,row_,column_):
+    button=tk.Button(button_frame,fg=foreground_color,bg=backgroundcolor_,command=lambda: show(char),bd=1,pady=1,font=("arial",30,"bold"),height=height_,width=width_,text=text_)
+    button.grid(row=row_,column=column_)
+
+
+# ---------------------------------Main program-------------------------------
+root = tk.Tk()  # creation of an object(root here)
+root.title("Simple Calculator")
+root.geometry("650x750+100+200") #Size of the window(570*600)
```

Comment:
use constante for the number "Width hight" and put is at the top of the file and declare it as global